client/src/pages/Profiles/Profiles.tsx

```diff
@@ -0,0 +1,109 @@
+import { Button, TextField, Typography } from '@mui/material';
+import useStyles from './useStyles';
+import hero from '../../images/landing/hero.jpg';
+import { useEffect, useState } from 'react';
+import { getProfiles } from '../../helpers/APICalls/getProfiles';
+
+export default function Profiles() {
+  const classes = useStyles();
+  const [profiles, setProfiles] = useState([]);
+
+  useEffect(() => {
+    getProfiles().then((resp) => setProfiles(resp.profiles));
```

Comment:
Let's update the snackbar here in case of a promise rejection (we would also want to chain a 'catch' block after the then to handle unexpected errors)